website/content/docs/release-notes/consul-k8s/v0_47_x.mdx

```diff
@@ -43,5 +43,5 @@ The changelogs for this major release version and any maintenance versions are l
 
 ~> **Note:** The following link takes you to the changelogs on the GitHub website.
 
-- [0.47.0](https://github.com/hashicorp/consul-k8s/releases/tag/v0.47.0)
+- [0.47.0](https://github.com/hashicorp/consul-k8s/releases/tag/v0.48.0)
```

Comment:
I think these changes are for a new `consul-k8s/v0_48_x.mdx` page. Is that right?

Reply:
Thanks I got myself confused these set of links are for 0.47.0, updated.